wiki/Glossary/en.md

```diff
@@ -231,16 +205,24 @@ However, do note that the skin's defined combo colours is only used when the map
 
 The combo fire was a feature in osu! that would display a burning yellow flame after obtaining a combo of 30.
 If the player obtains a combo of 500, the combo fire colour will change from yellow to blue.
-The combo fire will extinguish itself if the combo was broken (but would still be reobtainable).
+The combo fire will extinguish itself if the combo was broken but would still be reobtainable.
+
+This feature was disabled due to it being a suspect of lag spikes and having a bad appearance then was removed.
+
+### Cookie
 
-This feature was disabled due to it being a suspect of lag spikes (and having a bad appearance) then was removed.
+![](img/Intro_static.jpg)
+Yes, that big circle that reads "osu!" is the cookie
+
+The cookie, is another word for "osu!logo" which has appeared many times in the osu!client.
```

Comment:
"The cookie is typically used to describe the circular osu! logo which appears at the very start of the game, and refers to its circular shape."